src/nvim/buffer_defs.h

```diff
@@ -356,6 +356,7 @@ struct mapblock {
   char m_silent;                /* <silent> used, don't echo commands */
   char m_nowait;                /* <nowait> used */
   char m_expr;                  /* <expr> used, m_str is an expression */
+  char m_cmd;                  /* <command> used, m_str is a command */
```

Comment:
Misaligned. Also I guess this should be `bool m_cmd`. Or, actually, `enum { kMapTypeDirect, kMapTypeExpr, kMapTypeCommand } m_type` and not two flag variables which must not be both true at the same time.

And `///<` comments, everywhere.